src/CommunityToolkit.Maui.Core/Essentials/FileSaver/IFileSaver.shared.cs

```diff
@@ -12,6 +14,11 @@ public interface IFileSaver
 	/// <param name="fileName">File name with extension</param>
 	/// <param name="stream"><see cref="Stream"/></param>
 	/// <param name="cancellationToken"><see cref="CancellationToken"/></param>
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
```

Comment:
I'm not aware of any problems with this feature on iOS 13 and lower? Does this feature not work on iOS 13, for example? 

Let's remove those version numbers unless we have a known issue on iOS 13 and lower.

If there is a known issue, please open an Issue and comment here with the link.

```suggestion
	[SupportedOSPlatform("iOS")]
	[SupportedOSPlatform("MacCatalyst")]
```